src/StatusTransitionsHandler.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+namespace RebelCode\Bookings\WordPress\Module;
+
+use Dhii\Exception\CreateInvalidArgumentExceptionCapableTrait;
+use Dhii\I18n\StringTranslatingTrait;
+use Dhii\Invocation\InvocableInterface;
+use Dhii\Util\Normalization\NormalizeArrayCapableTrait;
+use Psr\EventManager\EventInterface;
+use Traversable;
+
+/**
+ * Handler for providing the status related state and functionality.
+ *
+ * @since [*next-version*]
+ */
+class StatusTransitionsHandler implements InvocableInterface
```

Comment:
What is the result of applying this handler to a request, in general?

Reply:
This question is about class name? I mean in this version this class name doesn't descriptive enough and in the new version I should fix it?

Reply:
I mean, there should be a description of what this class does to the event that it handles.